src/main/java/htsjdk/samtools/BinWithOffset.java

```diff
@@ -0,0 +1,21 @@
+package htsjdk.samtools;
+
+import htsjdk.samtools.Bin;
+
+public class BinWithOffset extends Bin {
+
+    private long lOffset;
```

Comment:
If there isn't any need for changing it I'd make this final and remove the setter.  Always simpler that way.